include/miopen/miopen.h

```diff
@@ -6094,7 +6094,7 @@ MIOPEN_EXPORT miopenStatus_t miopenT5LayerNormForward(miopenHandle_t handle,
  * @param sizeInBytes              Pointer to data to return the minimum workspace size
  * @return                         miopenStatus_t
  */
-extern "C" miopenStatus_t
+MIOPEN_EXPORT miopenStatus_t
```

Comment:
Why did you remove `extern "C"` specifier?

Reply:
Because the entire file is enclosed already. 
```
#ifdef __cplusplus__
extern "C" {
#endif
...
#ifdef __cplusplus__
}
#endif
```
There is no need for explicit `extern "C"`.